import_video.py

```diff
@@ -65,16 +62,16 @@ def get_params(self, path):
                 try:
                     lng = float(match.group('lng'))
                 except: pass
-        if not camera:
-            logging.warn('did not detect camera name, assuming "unnamed"')
-            camera = DEFAULT_CAMERA_NAME
+        if not camera_name:
+            logging.error('did not detect camera name, assuming "%s"', self.DEFAULT_CAMERA_NAME)
```

Comment:
missing:
camera_name = self.DEFAULT_CAMERA_NAME

and log camera_name rather than constant.